docs/docsite/rst/module_docs/azure_rm_routetable_module.rst

```diff
@@ -0,0 +1,459 @@
+.. _azure_rm_routetable:
```

Comment:
This file shouldn't be part of the PR. It is generated when we build our docs.

Reply:
Removed with newest commit